src/kernels/Conv_Winograd_v13_3_12_epilogue.inc

```diff
@@ -32,6 +32,11 @@ workgroup_size_x = 512
 .amdgcn.next_free_sgpr = 101
 .amdgcn.next_free_vgpr = 128
 
+//xnack disabled by default for asm kernels
```

Comment:
The changes like this are not necessary, but let's remove these later, when time permits.

Reply:
I mean that it's enough to change only one line.